src/share/poudriere/jail.sh

```diff
@@ -846,6 +987,22 @@ create_jail() {
 		# Do not check valid version given one can have a specific branch
 		FCT=install_from_vcs
 		;;
+	pkg=*)
+		FCT=install_from_pkg
+		PKGREPO="${METHOD##*=}"
+		[ -z "${PKGREPO}" ] && \
+		    err 1 "Must use format -m pkg=<url>/<to>/<repo>"
+		METHOD="${METHOD%%=*}"
+		;;
+	ports=*)
+		PTNAME="${METHOD#ports=}"
+	        # test if it already exists
+		porttree_exists ${PTNAME} || err 1 "No such ports name ${PTNAME}"
+		PORTS_BASE="${BASEFS:=/usr/local${ZROOTFS}}/ports/${PTNAME}"
+		test -d ${PORTS_BASE} || err 1 "No such ports directory"
```

Comment:
No need to call `test` and not the right style. `[ -d "${PORTS_BASE}" ] || err`

Reply:
This is fixed now, however I just pulled this style from the src= section a few lines below:

`test -d ${SRC_BASE} || err 1 "No such source directory"`

FYI in case you want to fix that one also later.